react-design/my-app/src/App.jsx

```diff
@@ -0,0 +1,23 @@
+// import { useState } from 'react'
+import RotatingBanner from './RotatingBanner';
+import './RotatingBanner.css';
+import './App.css';
+
+const items = [
+  'Aardvark',
+  'Bengal',
+  'Caterpillar',
+  'Dromedary',
+  'Elephant',
+  'Ferret',
+];
+
+function App() {
```

Comment:
Please complete step 3 of Phase 1. Add in the CSS necessary to match the mockup.

Reply:
The mockup is [here.](https://lms.learningfuze.com/code-exercises/Learning-Fuze/curriculum/react-design?v=db4a36affbd65e5be4fc5277069ee883bb0d4ec0) It just  should pretty much look like the final version, but with zero functionality!

Reply:
![Screenshot (172)](https://github.com/annaaksenov/m2-0723-code-solutions/assets/121647003/8d65a901-3f1c-48f3-9526-3a7a2004deac)
I am unsure what you mean, what part is not like the mockup?

Reply:
All good! It appears you attached the wrong screenshot! Let me quickly re-review!